tests/mapped_struct.py

```diff
@@ -855,11 +862,11 @@ def gen_values(self, n, reversed = False, shuffled = False, gen_dupes = False):
             r.shuffle(keys)
         if gen_dupes:
             return itertools.chain(
-                itertools.izip(keys, xrange(0, 2*n, 2)),
-                itertools.islice(itertools.izip(keys, xrange(0, 2*n, 2)), 10, None),
+                zip(keys, xrange(0, 2*n, 2)),
+                itertools.islice(zip(keys, xrange(0, 2*n, 2)), 10, None),
```

Comment:
This materializes the sequences in py2 and will use a lot of RAM, use izip in py2:

``` python
from future.builtins import zip
```

That requires future, but it's a good way to get conisstent behavior.

There's also `map` BTW.

Reply:
Hadn't seen it